code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2249,3 +2249,19 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	taste_description = "sweet corn beer and the hood life"
 	glass_name = "glass of malt liquor"
 	glass_desc = "A freezing pint of malt liquor."
+
+/datum/reagent/consumable/ethanol/coggernac
+	name = "COG-Nac"
+	description = "I don't even know what an eminence is, but I want him to recall."
+	metabolization_rate = INFINITY
+	boozepwr = 30
+	quality = DRINK_FANTASTIC
+	taste_description = "bronze and brass"
```

Comment:
bronze is fake brass, maybe "cogs and brass"

Reply:
done